qa/standalone/misc/availability.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
```

Comment:
Wasn't sure where this test should live so I added the test in the `misc` dir. Please let me know if there is a more appropriate path for it.